### PR TITLE
V13 downgrade + font awesome local serving

### PR DESCRIPTION
**Pull request by pietzschke**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ before_install:
 install:
   - npm install
 before_script:
-  - npm install -g @angular/cli
+  - npm install -g @angular/cli@13
```
Comment (philippesc) [inline .travis.yml:15]: Hardcoding this to version 13?

File: angular.json [hunk 2]
```diff
@@ -66,7 +66,12 @@
             "tsConfig": "projects/angular-editor-app/tsconfig.app.json",
             "assets": [
               "projects/angular-editor-app/src/favicon.ico",
-              "projects/angular-editor-app/src/assets"
+              "projects/angular-editor-app/src/assets",
```
Comment (philippesc) [inline angular.json:69]: This step is also needed to be add in README or something

File: projects/angular-editor-app/src/app/app.component.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import {Component, OnInit} from '@angular/core';
-import {AngularEditorConfig} from 'angular-editor';
+
 import {FormBuilder, FormGroup, Validators} from '@angular/forms';
+import { AngularEditorConfig } from 'projects/angular-editor/src/lib/config';
```
Comment (philippesc) [inline projects/angular-editor-app/src/app/app.component.ts:4]: Why don't we keep importing from the node module or define path in tsconfig?

File: projects/angular-editor/package.json [hunk 4]
```diff
@@ -10,8 +10,8 @@
     "url": "https://github.com/kolkov/angular-editor/issues"
   },
   "peerDependencies": {
-    "@angular/common": ">=14",
-    "@angular/core": ">=14"
+    "@angular/common": ">=13",
```
Comment (philippesc) [inline projects/angular-editor/package.json:13]: 🎉

File: projects/angular-editor/src/lib/angular-editor.component.ts [hunk 5]
```diff
@@ -205,7 +205,10 @@ export class AngularEditorComponent implements OnInit, ControlValueAccessor, Aft
    * Executed from the contenteditable section while the input property changes
    * @param element html element from contenteditable
    */
-  onContentChange(element: HTMLElement): void {
+  onContentChange(element: HTMLElement, target?: EventTarget): void {
+    if(target){
+      element = target as HTMLElement;
+    }
```
Comment (philippesc) [inline projects/angular-editor/src/lib/angular-editor.component.ts:211]: I don't understand this change.

File: .travis.yml (hunk 1 above)
Comment (pietzschke) [reply .travis.yml:15]: Buildpipeline is using latest otherwise

File: angular.json (hunk 2 above)
Comment (philippesc) [reply angular.json:69]: I added some description

File: projects/angular-editor-app/src/app/app.component.ts (hunk 3 above)
Comment (philippesc) [reply projects/angular-editor-app/src/app/app.component.ts:4]: Changed it backwards, Typescript path is working as expected.

File: projects/angular-editor/src/lib/angular-editor.component.ts (hunk 5 above)
Comment (philippesc) [reply projects/angular-editor/src/lib/angular-editor.component.ts:211]: I undid it

**Review by philippesc — APPROVED** [review]

**Pull request merged**